Solution1/hw_3 Task2/Program .cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace HW03.Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a1 = 32;
+            Int32 a2 = 32;
+           Console.WriteLine(a1.GetType());
+           Console.WriteLine(a2.GetType());
+        }
+    }
+}
```

Comment:
that needs to add all simple types, such as bool, double, char, string, object, etc.